modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/MiniCart.js

```diff
@@ -37,42 +37,35 @@ function MiniCart({
 	orderId,
 	spritemap,
 }) {
-	const AJAX = ServiceProvider.DeliveryCartAPI('v1');
-
 	const [isOpen, setIsOpen] = useState(false),
 		[isUpdating, setIsUpdating] = useState(false),
 		[cartState, updateCartState] = useState({}),
 		[actionURLs, setActionURLs] = useState(cartActionURLs),
 		[CartView, setCartView] = useState(null);
 
 	const closeCart = () => setIsOpen(false),
-		openCart = () => setIsOpen(true),
-		resetCartState = useCallback(() => updateCartState({}), [
-			updateCartState,
-		]);
-
-	// eslint-disable-next-line react-hooks/exhaustive-deps
-	const updateCartModel = ({orderId: cartId}) =>
-		AJAX.getCartByIdWithItems(cartId)
+		openCart = () => setIsOpen(true);
```

Comment:
Please try to move towards a one-statement-per-variable-declaration style in the future ([we'll be linting for it at some point](https://github.com/liferay/liferay-frontend-projects/issues/11)).